regen-network/regen-web
# David/team page tweaks

## Description

[Sanity PR](https://github.com/regen-network/regen-sanity/pull/17) (not required to merge this as it simply removes validation on description for team members, however should be done soon). 

- Update footer careers link
- Fix type errors in `MarketingNav` (which fixed the `active` state on dropdown items too 🥳 )
- remove description from team members

Closes: regen-network/regen-registry#959

---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...


### How to test

1. Go to homepage, click link in footer. Confirm it goes to notion doc
2. confirm no descriptions for team members

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...
